api-test/config/conf.js

```diff
@@ -0,0 +1,14 @@
+global.expect = require('chai').expect
+global.request = require('supertest')
+global.HttpStatus = require('http-status-codes')
+
+module.exports = {
+    BASE_URL: process.env.BASE_URL,
+    reporter: 'node_modules/mochawesome',
+    'reporter-option': [
+        'overwrite=true',
+        'reportTitle=My\ Custom\ Title',
+        'showPassed=false'
+    ],
+};
+
```

Comment:
All files in api-tests should not show in your PR as you've not added them. These should be what is in main branch.